Setup.sh

```diff
@@ -53,9 +53,26 @@ fi
 DBB_MODELER_APPCONFIG_DIR="$DBB_MODELER_WORK/modeler-configs"
 DBB_MODELER_APPLICATION_DIR="$DBB_MODELER_WORK/applications"
 DBB_MODELER_LOGS="$DBB_MODELER_WORK/logs"
-DBB_MODELER_METADATA_STORE_DIR="$DBB_MODELER_WORK/dbb-metadatastore"
 DBB_MODELER_DEFAULT_GIT_CONFIG="$DBB_MODELER_WORK/git-config"
 
+# Migration Modeler MetaDataStore configuration
+# Default value for the Metadatastore type - Valid values are "file" or "db2"
+DBB_MODELER_METADATASTORE_TYPE="file"
+# Default path for the File Metadatastore location
+DBB_MODELER_FILE_METADATA_STORE_DIR="$DBB_MODELER_WORK/dbb-metadatastore"
+# Default path for the DB2 Metadatastore Connection configuration file
+DBB_MODELER_DB2_METADATASTORE_CONFIG_FILE="$DBB_MODELER_WORK/db2Connection.conf"
+# DB2 User ID to connect through the JDBC driver
+DBB_MODELER_DB2_METADATASTORE_ID="user"
```

Comment:
Should we call it the `JDBC_USER` ?
```suggestion
DBB_MODELER_DB2_METADATASTORE_JDBCID="user"
```

Reply:
Changed